data/test_files/configurables/culture_graphics.txt

```diff
@@ -13,4 +11,7 @@ culture_1 = {
     army_portraits = {
         GFX_general_1
     }
+
+	graphical_culture = asian_gfx
+	graphical_culture_2d = asian_2d
```

Comment:
Looks like some parts of the file are tabs and some are spaces. Mind seeing if that can be auto-formatted?